specification/types.md

```diff
@@ -26,6 +26,10 @@ A numeric value of unspecified type or size. Implementation languages may furthe
 
 Structured data, presented however is idiomatic in the implementation language, such as JSON or YAML.
 
+### Collection
+
+A list or series of similar data types, presented however is idiomatic in the implementation language.
```

Comment:
Is this an ordered thing? Is it worth calling it out?

Reply:
This is a fairly generic definition since I didn't want to place too many constraints on the collection type returned per language. If order matters for a collection, I would think we would call that out in the requirement/section where it was applicable versus the blanket unordered callout. But I can add that if it's important to state here!